sdk/python/feast/feature_store.py

```diff
@@ -90,6 +90,10 @@
 warnings.simplefilter("once", DeprecationWarning)
 
 
+if TYPE_CHECKING:
+    from feast.embedded_go.online_features_service import EmbeddedOnlineFeatureServer
```

Comment:
just curious, why do we need to wrap this with `TYPE_CHECKING`?

Reply:
This enables lazy importing of `EmbeddedOnlineFeatureServer`. In case if somebody would need to run feast locally (set up from GH, eg) and fix something small w/o need to build go library. Not very strong case, but still could be useful.